Makefile

```diff
@@ -60,7 +60,10 @@ test-compile-changed: os-autoinst/
 test-yaml-valid:
 	$(eval YAMLS=$(shell sh -c "git ls-files schedule/ test_data/ | grep '\\.ya\?ml$$'"))
 	if test -n "$(YAMLS)"; then \
-      export PERL5LIB=${PERL5LIB_} ; tools/test_yaml_valid $(YAMLS);\
+      export PERL5LIB=${PERL5LIB_} ; \
+      for yaml_file in $(YAMLS); do \
```

Comment:
Maybe consider #13006 ? `xargs` should be much faster than calling yamllint on every file

Reply:
Trying:) Thanks for the assistance.

Reply:
Ok, since this was still failing, I experimented a bit more, and it seems the whole command is too long for make.
I found a working way for now:
```
test-yaml-valid:         
        $(eval YAMLS=$(shell sh -c "git ls-files schedule/ test_data/ | grep '\\.ya\?ml$$'"))
        if test -n "$(YAMLS)"; then \
                export PERL5LIB=${PERL5LIB_} ; echo "$(YAMLS)" | xargs tools/test_yaml_valid ;\
        else \
          echo "No yamls modified.";\
        fi                                                                                                                                                                                                                                   
        if test -n "$(YAMLS)"; then \
                which yamllint >/dev/null 2>&1 || echo "Command 'yamllint' not found, can not execute YAML syntax checks";\
                echo "$(YAMLS)" | xargs yamllint -c .yamllint;\
        fi                                                                                                            
```
This way the commands are split into 2 and a bit shorter.

Reply:
Added in scope of https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13006